### PR TITLE
add link to html, gpg sig, fix toc spacing

### PR DESCRIPTION
**Pull request by alorenzo175**
- [ ] Closes #xxxx .
  - [ x ] I am familiar with the [contributing guidelines](https://solarforecastarbiter-core.readthedocs.io/en/latest/contributing.html).
  - [ ] Tests added.
  - [ ] Updates entries to [`docs/source/api.rst`](https://github.com/SolarArbiter/solarforecastarbiter-core/blob/master/docs/source/api.rst) for API changes.
  - [ ] Adds descriptions to appropriate "what's new" file in [`docs/source/whatsnew`](https://github.com/SolarArbiter/solarforecastarbiter-core/tree/master/docs/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
  - [ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
  - [ x ] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

File: solarforecastarbiter/tests/data/report_metadata.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
             "mbe"
         ],
         "categories": [
-          "Total",
-          "Date",
-          "Hour of the day"
+          "total",
+          "date",
+          "hour"
```
Comment (wholmgren) [inline solarforecastarbiter/tests/data/report_metadata.json]: I'm surprised/concerned that the tests pass on master.

Comment (alorenzo175) [reply solarforecastarbiter/tests/data/report_metadata.json]: I think mocking interferes with the actual values

**Comment by alorenzo175** [comment]
I modified the test to not use report_metadata.json  since the values in it aren't used at all, but would like to keep the file for testing changes to the report manually with the cli

File: solarforecastarbiter/tests/data/report_metadata.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+    "_commant": "This file is not currently used for tests, but can be used with the CLI tool to generate a real report when using data from the prod. API.",
```
Comment (wholmgren) [inline solarforecastarbiter/tests/data/report_metadata.json]:
```suggestion
    "_comment": "This file is not currently used for tests, but can be used with the CLI tool to generate a real report when using data from the prod. API.",
```

**Pull request merged**